apps/verticalhome/js/stores/item.js

```diff
@@ -214,6 +216,27 @@
       } else {
         self.initSources(cb);
       }
+
+      // Open up the migration datastore before signalling readiness.
+      var successCallback = () => {
+        onsuccess && onsuccess(isEmpty);
+      };
+
+      navigator.getDataStores(DATASTORE_NAME).then((stores) => {
```

Comment:
Seems like this runs all the time? It should likely only run on IAC connection, from FTU or the system.